back_end/transaction/transaction_interface.go

```diff
@@ -0,0 +1,6 @@
+package transaction
+
+type ITransaction interface {
```

Comment:
Please, move the interface to back_end/merkle_tree/transactions_merkle_tree.go

Reply:
Impossible, this interface will be used for other things, block for instance

Reply:
Currently, there is no such need.